docs/api.md

```diff
@@ -669,6 +677,7 @@ stream.chain(f):   -1--2-13---2-1-233|
 
 Note the difference between [`concatMap`](#concatmap) and [`chain`](#chain): `concatMap` concatenates, while `chain` merges.
 
+<!-- skip-example -->
```

Comment:
I did indeed. I'm considering adding a timeout option to markdown-doctest so that we can test examples like these without unnecessarily waiting for completion. I'm also open to any ideas you have 😄